### ISSUE TITLE
Escopo e tarefas

### ISSUE THREAD

**Issue by lucascprazeres** [lucascprazeres/sof_guru#1]
## Primeiros passos
- [x] Receber pergunta via command arguments
- [x] Fazer puppeteer buscar pela pergunta no stack overflow
- [x] Extrair primeiro resultado e printar na tela
- [x] Criar tutorial de instalação do script
## Avançando um pouco mais...
- [x] Adicionar tratativa de sugestão principal
- [ ] Adicionar algum tipo de fallback, para quando o primeiro resultado não bastar
- [ ] Adicionar outras opções ao comando

**Comment by Daniel-Vinicius** [comment]
Olá Lucas, testei e adorei o seu projeto, porém não consegui uma forma de personalizar o texto que busca por algo diretamente na linha de comando, toda vez que eu executo o projeto ele retorna uma busca no google com o primeiro argumento assim
 _How can I check for “undefined” in JavaScript?_.
Sou Dev Javascript, Node e React, Eu gostaria de contribuir com o projeto, seja fazendo um README.MD mais completo ou colorindo as partes mais importantes na linha de comando kk. Até Hoje nunca usei o **puppeteer** mas tenho interesse.

**Comment by lucascprazeres** [comment]
> Olá Lucas, testei e adorei o seu projeto, porém não consegui uma forma de personalizar o texto que busca por algo diretamente na linha de comando, toda vez que eu executo o projeto ele retorna uma busca no google com o primeiro argumento assim
> _How can I check for “undefined” in JavaScript?_.
> Sou Dev Javascript, Node e React, Eu gostaria de contribuir com o projeto, seja fazendo um README.MD mais completo ou colorindo as partes mais importantes na linha de comando kk. Até Hoje nunca usei o **puppeteer** mas tenho interesse.

Sim! Eu esqueci completamente de ensinar a configurar o script no ambiente. Já já subo uma atualização.
Obrigado pelo feedback!!

**Comment by Daniel-Vinicius** [comment]
Olá Lucas Obrigado pela Atualização, agora funciona bem, o que acontece é que no windows não se pode passar parâmetros por arquivos bat, então eu peguei a versão antiga e criei um atalho do windows para a pasta bin e lá eu mesmo abro o terminal e executo yarn start
